### PR TITLE
Adding the Perk60 project, a 60% per key rgb PCB

### PR DESCRIPTION
**Pull request by 4pplet**
## Description

Adding the Perk60 project, a 60% per key rgb PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

Thanks! <3

### CODE REVIEW

File: keyboards/4pplet/perk60_iso/rev_a/chconf.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ *    `qmk chibios-confmigrate -i keyboards/acheron/austin/chconf.h -r platforms/chibios/common/configs/chconf.h`
+ */
+
+#pragma once
+
+#define CH_CFG_ST_FREQUENCY 10000
+
+#define CH_CFG_OPTIMIZE_SPEED FALSE
+
+#define CH_CFG_USE_CONDVARS_TIMEOUT FALSE
```
Comment (drashna) [inline keyboards/4pplet/perk60_iso/rev_a/chconf.h:28]: Is there a specific reason for these options?

File: keyboards/4pplet/perk60_iso/rev_a/rev_a.c [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    }, {
+        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+        8, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+        1, 1, 1, 4, 1, 1, 1, 1
+    }
+};
+
+void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
```
Comment (drashna) [inline keyboards/4pplet/perk60_iso/rev_a/rev_a.c:116]: This should be the kb level functino, or it should be weak. 
```suggestion
__attribute__((weak)) void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/4pplet/perk60_iso/rev_a/chconf.h (hunk 1 above)
Comment (4pplet) [reply keyboards/4pplet/perk60_iso/rev_a/chconf.h:28]: I assumed these were needed following the joker template and gondos recommended code to check out. Tested the PCB without the chconf.h and all works well. I'll remove it.

Thanks!

Comment (drashna) [reply keyboards/4pplet/perk60_iso/rev_a/chconf.h]: That's exactly why I asked.  I ... did the same. :D   And actually for me, removing these settings improved performance on the board.

**Review by drashna — APPROVED** [review]

File: keyboards/4pplet/perk60_iso/rev_a/rules.mk [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+# Build Options
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no   # Mouse keys
+EXTRAKEY_ENABLE = yes   # Audio control and System control
+CONSOLE_ENABLE = no    # Console for debug
+COMMAND_ENABLE = no    # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no
+RGBLIGHT_ENABLE = no
```
Comment (fauxpark) [inline keyboards/4pplet/perk60_iso/rev_a/rules.mk:15]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/4pplet/perk60_iso/rev_a/rules.mk [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# MCU name
+MCU = STM32F411
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Build Options
```
Comment (fauxpark) [inline keyboards/4pplet/perk60_iso/rev_a/rules.mk:7]:
```suggestion
# Build Options
#   change yes to no to disable
#
```

File: keyboards/4pplet/perk60_iso/rev_a/rev_a.c [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "rev_a.h"
+
+const is31_led PROGMEM g_is31_leds[DRIVER_LED_TOTAL] = {
```
Comment (fauxpark) [inline keyboards/4pplet/perk60_iso/rev_a/rev_a.c:19]:
```suggestion
#ifdef RGB_MATRIX_ENABLE
const is31_led PROGMEM g_is31_leds[DRIVER_LED_TOTAL] = {
```

File: keyboards/4pplet/perk60_iso/rev_a/rev_a.c [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+__attribute__((weak)) void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
+    if (host_keyboard_led_state().caps_lock && CAPS_LOCK_ENABLE) {
+        for (uint8_t i = led_min; i <= led_max; i++) {
+            if (g_led_config.flags[i] & CAPS_LED_GROUP) {
+                rgb_matrix_set_color(i, CAPS_LOCK_COLOR);
+            }
+        }
+    }
+}
+
```
Comment (fauxpark) [inline keyboards/4pplet/perk60_iso/rev_a/rev_a.c:125]:
```suggestion
#endif
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by 4pplet** [comment]
Thanks for the input! @drashna @fauxpark , much appreciated <3

**Comment by 4pplet** [comment]
This should be ready for merge now? :)

**Comment by 4pplet** [comment]
Sorry for the ping @fauxpark , I believe this is ready for merge now? :)

File: keyboards/4pplet/perk60_iso/rev_a/config.h [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_WIDE
+#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_CROSS
+#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_NEXUS
+#define ENABLE_RGB_MATRIX_SPLASH
+#define ENABLE_RGB_MATRIX_SOLID_SPLASH
+
+/* Allow VIA to edit lighting */
+#ifdef VIA_ENABLE
+#    define VIA_QMK_RGBLIGHT_ENABLE
+#endif
```
Comment (drashna) [inline keyboards/4pplet/perk60_iso/rev_a/config.h:83]: Will no longer be needed at the end of the month.
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/4pplet/perk60_iso/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+// main layer
+[0] = LAYOUT_all(
+  KC_ESC,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,       KC_6,      KC_7,      KC_8,      KC_9,      KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
+  KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,       KC_Y,      KC_U,      KC_I,      KC_O,      KC_P,    KC_LBRC, KC_RBRC, KC_ENT,
+  KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,       KC_H,      KC_J,      KC_K,      KC_L,      KC_SCLN, KC_QUOT, KC_NUHS,
```
Comment (noroadsleft) [inline keyboards/4pplet/perk60_iso/keymaps/default/keymap.c:24]:
```suggestion
[0] = LAYOUT_60_iso(
  KC_ESC,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,       KC_6,      KC_7,      KC_8,      KC_9,      KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
  KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,       KC_Y,      KC_U,      KC_I,      KC_O,      KC_P,    KC_LBRC, KC_RBRC,
  KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,       KC_H,      KC_J,      KC_K,      KC_L,      KC_SCLN, KC_QUOT, KC_NUHS, KC_ENT,
```

File: keyboards/4pplet/perk60_iso/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+  KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,     KC_C,     KC_V,       KC_B,      KC_N,      KC_M,      KC_COMM,   KC_DOT,  KC_SLSH, KC_RSFT,
+  KC_LCTL,  KC_LGUI,  KC_LALT,                                  KC_SPC,                                     KC_RALT,   KC_RGUI, MO(1), KC_RCTL),
+  // basic function layer
+[1] = LAYOUT_all(
+  RESET,    KC_F1,    KC_F2,    KC_F3,    KC_F4,     KC_F5,     KC_F6,      KC_F7,          KC_F8,              KC_F9,      KC_F10,  KC_F11,   KC_F12,  KC_DEL,
+  KC_TRNS,  KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS,        KC_TRNS,            KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,
+  KC_TRNS,  KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS,        KC_TRNS,            KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS,
```
Comment (noroadsleft) [inline keyboards/4pplet/perk60_iso/keymaps/default/keymap.c:31]:
```suggestion
[1] = LAYOUT_60_iso(
  RESET,    KC_F1,    KC_F2,    KC_F3,    KC_F4,     KC_F5,     KC_F6,      KC_F7,          KC_F8,              KC_F9,      KC_F10,  KC_F11,   KC_F12,  KC_DEL,
  KC_TRNS,  KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS,        KC_TRNS,            KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS,
  KC_TRNS,  KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS,        KC_TRNS,            KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,
```

File: keyboards/4pplet/perk60_iso/keymaps/via/keymap.c [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+*/
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+// main layer// main layer
+[0] = LAYOUT_all(
+  KC_ESC,   KC_1,    KC_2,     KC_3,     KC_4,     KC_5,       KC_6,      KC_7,      KC_8,      KC_9,       KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
+  KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,       KC_Y,      KC_U,      KC_I,      KC_O,       KC_P,    KC_LBRC, KC_RBRC, KC_ENT,
+  KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,       KC_H,      KC_J,      KC_K,      KC_L,       KC_SCLN, KC_QUOT, KC_NUHS,
```
Comment (noroadsleft) [inline keyboards/4pplet/perk60_iso/keymaps/via/keymap.c:25]:
```suggestion
[0] = LAYOUT_60_iso(
  KC_ESC,   KC_1,    KC_2,     KC_3,     KC_4,     KC_5,       KC_6,      KC_7,      KC_8,      KC_9,       KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
  KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,       KC_Y,      KC_U,      KC_I,      KC_O,       KC_P,    KC_LBRC, KC_RBRC,
  KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,       KC_H,      KC_J,      KC_K,      KC_L,       KC_SCLN, KC_QUOT, KC_NUHS, KC_ENT,
```

File: keyboards/4pplet/perk60_iso/keymaps/via/keymap.c [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+  KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,     KC_C,     KC_V,       KC_B,      KC_N,      KC_M,      KC_COMM,    KC_DOT,  KC_SLSH, KC_RSFT,
+  KC_LCTL,  KC_LGUI,  KC_LALT,                           KC_SPC,                                 KC_RALT,      KC_RGUI, MO(1), KC_RCTL),
+  // basic function layer
+[1] = LAYOUT_all(
+  RESET,    KC_F1,    KC_F2,    KC_F3,    KC_F4,     KC_F5,     KC_F6,      KC_F7,          KC_F8,              KC_F9,      KC_F10,  KC_F11,   KC_F12,  KC_DEL,
+  KC_TRNS,  KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS,        KC_TRNS,            KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,
+  KC_TRNS,  KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS,        KC_TRNS,            KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS,
```
Comment (noroadsleft) [inline keyboards/4pplet/perk60_iso/keymaps/via/keymap.c:32]:
```suggestion
[1] = LAYOUT_60_iso(
  RESET,    KC_F1,    KC_F2,    KC_F3,    KC_F4,     KC_F5,     KC_F6,      KC_F7,          KC_F8,              KC_F9,      KC_F10,  KC_F11,   KC_F12,  KC_DEL,
  KC_TRNS,  KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS,        KC_TRNS,            KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS,
  KC_TRNS,  KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS,        KC_TRNS,            KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,
```

File: keyboards/4pplet/perk60_iso/keymaps/via/keymap.c [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+  KC_TRNS,  RGB_TOG,  RGB_MOD,  RGB_HUI,  RGB_SAI,   RGB_VAI,   RGB_SPI,    RGB_MODE_PLAIN, RGB_MODE_BREATHE,   RGB_MODE_RAINBOW,    KC_TRNS, KC_TRNS,  KC_TRNS,
+  KC_TRNS,  KC_TRNS,  KC_TRNS,                       KC_TRNS,                                                   KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS),
+
+[2] = LAYOUT_all(
+  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,      KC_TRNS,  KC_TRNS,   KC_TRNS,  KC_TRNS,
+  KC_TRNS,  KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,
+  KC_TRNS,  KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS,
```
Comment (noroadsleft) [inline keyboards/4pplet/perk60_iso/keymaps/via/keymap.c:39]:
```suggestion
[2] = LAYOUT_60_iso(
  KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,      KC_TRNS,  KC_TRNS,   KC_TRNS,  KC_TRNS,
  KC_TRNS,  KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS,
  KC_TRNS,  KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,
```

File: keyboards/4pplet/perk60_iso/keymaps/via/keymap.c [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS,
+  KC_TRNS,  KC_TRNS,  KC_TRNS,                        KC_TRNS,                            KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS),
+
+[3] = LAYOUT_all(
+  KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,      KC_TRNS,  KC_TRNS,   KC_TRNS,  KC_TRNS,
+  KC_TRNS,  KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,
+  KC_TRNS,  KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS,
```
Comment (noroadsleft) [inline keyboards/4pplet/perk60_iso/keymaps/via/keymap.c:46]:
```suggestion
[3] = LAYOUT_60_iso(
  KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,     KC_TRNS,      KC_TRNS,  KC_TRNS,   KC_TRNS,  KC_TRNS,
  KC_TRNS,  KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS,
  KC_TRNS,  KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,    KC_TRNS, KC_TRNS,  KC_TRNS, KC_TRNS,
```

File: keyboards/4pplet/perk60_iso/rev_a/rev_a.h [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+#define CAPS_LOCK_ENABLE true
+#define CAPS_LOCK_COLOR RGB_RED
+#define CAPS_LED_GROUP LED_FLAG_INDICATOR // change what leds to target, for example LED_FLAG_KEYLIGHT for alpas or LED_FLAG_MODIFIER for modifiers
+
+#include "quantum.h"
+
+#define LAYOUT_all( \
+    k00, k10, k01, k11, k02, k12, k03, k13, k04, k14, k05, k15, k06, k36, \
+    k20, k30, k21, k31, k22, k32, k23, k33, k24, k34, k25, k35, k26, k76,\
+    k40, k50, k41, k51, k42, k52, k43, k53, k44, k54, k45, k55, k46,\
```
Comment (noroadsleft) [inline keyboards/4pplet/perk60_iso/rev_a/rev_a.h:28]:
```suggestion
#define LAYOUT_60_iso( \
    k00, k10, k01, k11, k02, k12, k03, k13, k04, k14, k05, k15, k06, k36, \
    k20, k30, k21, k31, k22, k32, k23, k33, k24, k34, k25, k35, k26,      \
    k40, k50, k41, k51, k42, k52, k43, k53, k44, k54, k45, k55, k46, k76, \
```

File: keyboards/4pplet/perk60_iso/info.json [hunk 15]
```diff
@@ -0,0 +1,10 @@
...
...Lock", "x":0, "y":2, "w":1.75}, {"label":"A", "x":1.75, "y":2}, {"label":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":6.75, "y":2}, {"label":"J", "x":7.75, "y":2}, {"label":"K", "x":8.75, "y":2}, {"label":"L", "x":9.75, "y":2}, {"label":":", "x":10.75, "y":2}, {"label":"@", "x":11.75, "y":2}, {"label":"~", "x":12.75, "y":2}, {"label":"Shift", "x":0, "y":3, "w":1.25}, {"label":"|", "x":1.25, "y":3}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"Shift", "x":12.25, "y":3, "w":2.75}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4, "w":1.25}, {"label":"Alt", "x":2.5, "y":4, "w":1.25}, {"x":3.75, "y":4, "w":6.25}, {"label":"AltGr", "x":10, "y":4, "w":1.25}, {"label":"Win", "x":11.25, "y":4, "w":1.25}, {"label":"Menu", "x":12.5, "y":4, "w":1.25}, {"label":"Ctrl", "x":13.75, "y":4, "w":1.25}]
```
Comment (noroadsleft) [inline keyboards/4pplet/perk60_iso/info.json:7]:
```suggestion
        "LAYOUT_60_iso": {
            "layout": [{"label":"\u00ac", "x":0, "y":0}, {"label":"!", "x":1, "y":0}, {"label":"\"", "x":2, "y":0}, {"label":"\u00a3", "x":3, "y":0}, {"label":"$", "x":4, "y":0}, {"label":"%", "x":5, "y":0}, {"label":"^", "x":6, "y":0}, {"label":"&", "x":7, "y":0}, {"label":"*", "x":8, "y":0}, {"label":"(", "x":9, "y":0}, {"label":")", "x":10, "y":0}, {"label":"_", "x":11, "y":0}, {"label":"+", "x":12, "y":0}, {"label":"Backspace", "x":13, "y":0, "w":2}, {"label":"Tab", "x":0, "y":1, "w":1.5}, {"label":"Q", "x":1.5, "y":1}, {"label":"W", "x":2.5, "y":1}, {"label":"E", "x":3.5, "y":1}, {"label":"R", "x":4.5, "y":1}, {"label":"T", "x":5.5, "y":1}, {"label":"Y", "x":6.5, "y":1}, {"label":"U", "x":7.5, "y":1}, {"label":"I", "x":8.5, "y":1}, {"label":"O", "x":9.5, "y":1}, {"label":"P", "x":10.5, "y":1}, {"label":"{", "x":11.5, "y":1}, {"label":"}", "x":12.5, "y":1}, {"label":"Caps Lock", "x":0, "y":2, "w":1.75}, {"label":"A", "x":1.75, "y":2}, {"label":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":6.75, "y":2}, {"label":"J", "x":7.75, "y":2}, {"label":"K", "x":8.75, "y":2}, {"label":"L", "x":9.75, "y":2}, {"label":":", "x":10.75, "y":2}, {"label":"@", "x":11.75, "y":2}, {"label":"~", "x":12.75, "y":2}, {"label":"Enter", "x":13.75, "y":1, "w":1.25, "h":2}, {"label":"Shift", "x":0, "y":3, "w":1.25}, {"label":"|", "x":1.25, "y":3}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"Shift", "x":12.25, "y":3, "w":2.75}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4, "w":1.25}, {"label":"Alt", "x":2.5, "y":4, "w":1.25}, {"x":3.75, "y":4, "w":6.25}, {"label":"AltGr", "x":10, "y":4, "w":1.25}, {"label":"Win", "x":11.25, "y":4, "w":1.25}, {"label":"Menu", "x":12.5, "y":4, "w":1.25}, {"label":"Ctrl", "x":13.75, "y":4, "w":1.25}]
```

File: keyboards/4pplet/perk60_iso/rev_a/rules.mk [hunk 16]
```diff
@@ -0,0 +1,23 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = IS31FL3733
+
+# Enter lower-power sleep mode when on the ChibiOS idle thread
```
Comment (noroadsleft) [inline keyboards/4pplet/perk60_iso/rev_a/rules.mk:22]:
```suggestion
LAYOUTS = 60_iso

# Enter lower-power sleep mode when on the ChibiOS idle thread
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Moving the Enter key's matrix position identifier to the home row makes this keyboard compatible with QMK's `60_iso` Community Layout.

**Comment by 4pplet** [comment]
Thanks for the suggested changes @noroadsleft ! <3 Very good

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**